src/js/charts/line.js

```diff
@@ -117,6 +119,14 @@
           .attr('d', plot.area(args.data[which_line]))
           .attr('fill', args.colors === null ? '' : args.colors[line_id - 1])
           .attr('clip-path', 'url(#mg-plot-window-' + mg_target_ref(args.target) + ')');
+
+        svg.append('path')
+          .classed('mg-main-area', true)
+          .classed('mg-area' + line_id, true)
+          .classed('mg-area' + '-color', args.colors === null)
+          .attr('d', plot.area(args.data[which_line]))
+          .attr('fill', args.colors === null ? '' : args.colors[line_id - 1])
+          .attr('clip-path', 'url(#mg-plot-window-' + mg_target_ref(args.target) + ')');
```

Comment:
Could you explain what this is supposed to be doing? It doesn't seem correct to me (again depending on args when we're trying to add a "default" style which doesn't depend on them).